cli/src/modules/account.rs

```diff
@@ -84,6 +84,8 @@ impl Account {
                                 "account import mnemonic multisig [additional keys]",
                                 "Import mnemonic and additional keys for a multisig account",
                             ),
+                            ("account import bip32-watch", "Import a extended public key for a watch-only bip32 account"),
```

Comment:
Why not use `account watch bip32` and `account watch multisig` here?

Reply:
`bip32-watch` is the account type (it is a `FromStr` - not a command).  If I remember correctly, multisig doesn't need watch accounts. @biryukovmaxim @1bananagirl

Reply:
Discussion source of the reason here: https://github.com/kaspanet/rusty-kaspa/pull/470

Reply:
@coderofstuff yes we can rename import to watch !

Reply:
Added in PR  https://github.com/aspectron/rusty-kaspa/pull/92 @coderofstuff @aspect